python/ray/dataframe/test/test_dataframe.py

```diff
@@ -597,24 +601,45 @@ def test_ge():
 
 
 def test_get():
-    ray_df = create_test_dataframe()
+    df = pd.DataFrame({'col1': [0, 1, 2, 3],
```

Comment:
Prefer to convert test to `@pytest.fixture` and call from above. Feel free to create a new test with some string data if you would like.